### PR TITLE
Initial commit for 244

### PR DESCRIPTION
**Pull request by ibenitamar**

### CODE REVIEW

File: RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingCalendar/mobileAppointmentBookingCalendar.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { LightningElement, api, track } from "lwc";
 import customLabels from "./labels";
 
-export default class MobileAppointmentBookingRebookingCalendar extends LightningElement {
+export default class AaRebookingCalendar extends LightningElement {
```
Comment (maliroteh-sf) [inline RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingCalendar/mobileAppointmentBookingCalendar.js:4]: Can we rename `AaRebookingCalendar` to something more meaningful please?

File: RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingSettingsContainer/mobileAppointmentBookingSettingsContainer.js [hunk 2]
```diff
@@ -44,6 +43,7 @@ export default class MobileAppointmentBookingSettingsContainer extends Lightning
   ];
 
   connectedCallback() {
+    //this.showExactArrivalTime = true;
```
Comment (maliroteh-sf) [inline RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingSettingsContainer/mobileAppointmentBookingSettingsContainer.js:46]: nit: remove commented code

**Review by maliroteh-sf — COMMENTED** [review]
Your PRs are often very large. In the future please consider breaking the work across a few smaller PRs rather than one giant PR.... this will result in a faster and more effective/accurate code review.

Looking at the changes in this PR I can't tell whether you are using any APIs that are only available on 244, but judging from the title of the PR I assume this may be the case? If so then perhaps we should not take this PR until after 244 is shipped (since this repo is public and open to all)? @khawkins what are your thoughts on this?

File: RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingCalendar/mobileAppointmentBookingCalendar.js (hunk 1 above)
Comment (ibenitamar) [reply RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingCalendar/mobileAppointmentBookingCalendar.js]: @maliroteh-sf this is accidental I reverted this change

**Comment by ibenitamar** [comment]
@maliroteh-sf this would be the last modification for this component, only features that we didn't manage to complete in 242. I though I couldn't commit during the version since this git is open to use..

The  PR contains all the work that is left for this feature and was done during 244 work period, as far as I know there is no use of api that's specific to 244 and pending on future releases. More than that we were working and testing on a sandbox env (I can provide instances info in a private msg)..It's mostly UX screens and some bug fixes.

File: RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingResourceAssignment/mobileAppointmentBookingResourceAssignment.html [hunk 3]
```diff
@@ -7,16 +7,16 @@
...
             <template lwc:else>
-                <div>
+                <div class="resource-assignment-container">
                     <lightning-radio-group name="resourceAssignment"
                         aria-label={resourceAssignmentTitle}
                         options={options}
                         value={value}
                         type="radio"
                         onchange={handleAssignmentMethodChange}>
-                    </lightning-radio-group>
```
Comment (maliroteh-sf) [inline RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingResourceAssignment/mobileAppointmentBookingResourceAssignment.html:17]: nit: fix indentation

**Review by maliroteh-sf — APPROVED** [review]
Just a few very minor comments but overall LGTM 👍

**Pull request merged**